articles/cosmos-db/nosql/troubleshoot-bad-request.md

```diff
@@ -47,6 +47,16 @@ A response with this error means you are executing an operation and passing a pa
 ### Solution
 Send the partition key value parameter that matches the document property value.
 
+## <a name="partition-key-value-precision-loss"></a>Numeric partition key value precision loss
```

Comment:
Remove anchor, the title itself already is an anchor. Other titles have it because their names changed over time

Reply:
```suggestion
## Numeric partition key value precision loss
```